microsoftgraph/microsoft-graph-explorer-v4
# Fix: Snippet error messaging

## Overview
Closes #2429 

### Demo

Optional. Screenshots, `curl` examples, etc.

## Testing Instructions

* How to test this PR
Add any url to the query URL textbox then click on the code snippets tab
Notice that the correct error message is displayed
